example.rb

```diff
@@ -152,6 +189,8 @@ def print_message_sent(message_response)
   sender_id: SENDER_ID, recipient_number: RECIPIENT_NUMBER, document_id: "1234", caption: "Ignacio Chiazzo"
 )
 
+binding.pry
```

Comment:
@ignacio-chiazzo forget?

Reply:
Good catch.  I'm currently working on adding Templates API https://github.com/ignacio-chiazzo/ruby_whatsapp_sdk/pull/90; and I will update this PR once I ship the other. I will ping you for review on the other once ready.

Thanks for the review!